### DISCUSSION TITLE
"similar modules" models

### DISCUSSION THREAD

**Discussion by slingr-dev** [revk/xESP32-Faikout#39] [Q&A] [answered]
First off, apologies for messing with the Amazon stock by cancelling two orders, I just wanted to ensure to be rid of any assumptions so I don't end up with a few units unusable to me.

"This is code and PCB design to run on an ESP32 module and connect to a Daikin aircon unit in place of a BRP069B41, BRP069C41, or **similar modules**."

Do you think "similar modules" would include BRP072A43? I am working with 1 x FTXS24LVJU and 2 x CTXS07LVJU which both list BRP072A43 as their wireless module on the older datasheets before moving over to the Airzone units. If the ESP32-Faikin is compatible with BRP072A43 I can re-place my order (for a final time). I'm looking at https://github.com/unixko/DaikinCon for information on building the S21 leads.

Appreciate the response,
slingr

**Comment by revk** [comment]
Basically I have no way to tell other than trying it, sorry. I think Daikin S21 is pretty "standard", though not a published standard so very much reverse engineered. If the air-con has an S21 port it is likely to work.

**Comment by joshbenner** [comment, marked as answer]
I am using these PCBs on FTXS12LVJU (where I _previously_ used BRP072A43) and CTXS07LVJU, and they are working great.

**Comment by joshbenner** [comment]
@slingr-dev I've built my own S21 leads by ordering EHR-5 plugs and accompanying SEH-001T-P0.6 pins from DigiKey, along with a length of 4 conductor 22 AWG cable. I had to also get a JST crimper. The crimping was a little awkward to figure out at first, but once I got it down, it wasn't too bad.

In one of my units, I experimentally used the pre-made SEH-001T-P0.6 jumper wires and enclosed the Faikin PCB inside the unit's cover. This worked in one case, but it blocked Wifi signal too significantly in other cases.

**Reply by slingr-dev** [reply]
Thank you so much for the quality info @joshbenner - extremely appreciative
Also thanks @revk for this entire effort and project.

**Comment by slingr-dev** [comment]
@revk FTXS24LVJU is confirmed working. Unable to edit the wiki myself.

**Reply by revk** [reply]
Silly question? But how do I allow the wiki to be editable? I thought that was the whole point of a wiki!

I'll add that one though.

**Reply by revk** [reply]
OK I have allowed wiki edits!
